config/configv2_test.go

```diff
@@ -36,77 +36,84 @@ func TestDecodeConfig(t *testing.T) {
 	}
 
 	// Verify receivers
-	assert.Equal(t, len(config.Receivers), 2, "Incorrect receivers count")
+	assert.Equal(t, 2, len(config.Receivers), "Incorrect receivers count")
```

Comment:
switching param order to match expected and actual? nice

Reply:
Yes, to ensure proper error messages when tests fail (otherwise they are confusing).